### PR TITLE
Add BodyParserOptionsToken for bodyparser

### PR DESCRIPTION
**Pull request by flyingsky**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -85,6 +85,7 @@ const plugin: RPCPluginType =
     deps: {
       emitter: UniversalEventsToken,
       handlers: RPCHandlersToken,
+      bodyParserOptions: BodyParserOptionsToken,
```
Comment (ganemone) [inline src/server.js]: We should make this optional. If you change it to `BodyParserOptionsToken.optional` it will do the trick

Comment (ganemone) [reply src/server.js]: It would be great if we could get a test case also

File: src/server.js [hunk 2]
```diff
@@ -97,7 +98,7 @@ const plugin: RPCPluginType =
 
     middleware: deps => {
       const {emitter, handlers} = deps;
-      const parseBody = bodyparser();
+      const parseBody = bodyparser(bodyParserOptions);
```
Comment (KevinGrandon) [inline src/server.js]: This should be destructured from `deps` above.

File: src/server.js (hunk 1 above)
Comment (flyingsky) [reply src/server.js]: updated.

**Pull request merged**